haas/cli.py

```diff
@@ -247,17 +247,24 @@ def serve_networks():
     """Start the HaaS networking server"""
     from haas import model, deferred
     from time import sleep
+    import ConfigParser
     server.init()
     server.register_drivers()
     server.validate_state()
     model.init_db()
     migrations.check_db_schema()
+    # Check if config contains usable sleep_time
+    try:
+        sleep_time = float(cfg.get('devel', 'sleep_time'))
+    except (ConfigParser.NoSectionError, ConfigParser.NoOptionError,
+            ValueError):
+        sleep_time = 2
```

Comment:
I'm actually inclined to just not have an upper bound. The reason is just that it's extra stuff that needs to be documented, and therefore stuff a sysadmin has to sift through, and it's not checking any particularly important invariants -- it's just an arbitrary cap. I'd just check that it's positive and leave it at that.